Alloy/commands/compile/ast/handle-alloy-globals.js

```diff
@@ -0,0 +1,98 @@
+const toCheck = [ 'Alloy', '_', 'Backbone' ];
+var template = require('@babel/template').default;
+
+var buildRequire = template(`
+	var VARIABLE = REQUIRECALL;
+`);
+module.exports = function(babel) {
+	return {
+		name: 'app.js top level variables global transform',
+		visitor: {
+			CallExpression: function (path, state) {
+				const node = path.node;
+				if (node.callee.name !== 'require') {
+					return;
+				}
+				if (!node.arguments || !node.arguments[0]) {
+					return;
+				}
+				switch (node.arguments[0].value) {
+					case '/alloy':
+						this.imported.push('Alloy');
+						break;
+					case '/alloy/underscore':
+						this.imported.push('_');
+						break;
+					case '/alloy/backbone':
+						this.imported.push('Backbone');
+						break;
+				}
+			},
+			ImportDeclaration (path) {
+				const node = path.node;
+				if (!node.source || !node.source.value) {
+					return;
+				} 
+				switch (node.source.value) {
+					case '/alloy':
+						this.imported.push('Alloy');
+						break;
+					case '/alloy/underscore':
+						this.imported.push('_');
+						break;
+					case '/alloy/backbone':
+						this.imported.push('Backbone');
+						break;
+				}
+			},
+			ReferencedIdentifier(path) {
+				const node = path.node;
```

Comment:
instead assign `const name = path.node.name;` and refer to that, rather than hold the node and keep asking for it's name property in all the references.

Reply:
Done, though about using object destructing here but didn't want to play around with the eslint config. (which reminds me to get round to finally finishing moving alloy to eslint-config-axway soon)